steps/homeSteps.ts

```diff
@@ -0,0 +1,41 @@
+import { Locator, Page } from "@playwright/test";
+import { BaseSteps } from "./baseSteps";
+import { HomePage } from "../pages/homePage";
+
+export class HomeSteps extends BaseSteps {
+  private homePage: HomePage;
+
+  constructor(page: Page) {
+    super(page);
+    this.homePage = new HomePage(page);
+  }
+  async getUserByUserName(userNameValue: string): Promise<Locator> {
+    const users: Locator[] = await this.page.getByTestId("td-UserName").all();
+
+    for (const user of users) {
+      if ((await user.innerText()) === userNameValue) {
+        return user.locator("xpath=//parent::tr");
+      }
+    }
+
+    throw new Error(`Created user "${userNameValue}" was not found`);
```

Comment:
it will be better to write this message: 

User with "${userNameValue}" name is not found